core/js/src/typespecs/app_types.ts

```diff
@@ -158,6 +159,16 @@ export const datasetSchema = z
   .openapi("Dataset");
 export type Dataset = z.infer<typeof datasetSchema>;
 
+export const promptSchema = z.object({
+  id: z.string().uuid(),
```

Comment:
Should we include documentation for these fields, as we do in the other API objects? That is one benefit to using the `generateBaseTableSchema` helper function, because it autogenerates docs for the common fields.